communication/utils.py

```diff
@@ -28,19 +28,17 @@ def get_autologin_link(unique_token):
 class SmsSender:
 
     def send(self, send_to, send_from, message):
-        url = "https://junebug.qa.dig-it.me/jb/channels/ab79e9c5-7d15-440f-8bd0-6a0e2d2445b1/messages/"
+        url = settings.JUNEBUG_BASE_URL + settings.JUNEBUG_CHANNEL_ID
 
         payload = "{\"to\": \"%s\", \"from\": \"%s\", \"content\": \"%s\"}" % (send_to, send_from, message)
         headers = {
-            'authorization': "Basic anVuZWJ1ZzpEYkFPVEJzbTR6WGJVMTB4",
+            'authorization': settings.JUNEBUG_AUTHORISATION_TYPE,
```

Comment:
We should rather use the `JUNEBUG_USERNAME` and `JUNEBUG_PASSWORD` to generate the header, rather than requiring the auth to be specified in two different ways, this leads to easily creating an incorrect config.

`requests` has an `auth` argument that we can use for basic auth.